app/controllers/CyaMaintainProfileController.scala

```diff
@@ -47,6 +48,37 @@ class CyaMaintainProfileController @Inject() (
   private val errorMessage: String = "Unable to update Trader profile."
   private val continueUrl: Call    = routes.ProfileController.onPageLoad()
 
+  def onPageLoadNirmsNumber(): Action[AnyContent] = (identify andThen getData andThen requireData).async {
+    implicit request =>
+      traderProfileConnector.getTraderProfile(request.eori).flatMap { _ =>
+        request.userAnswers.get(NirmsNumberUpdatePage) match {
+          case Some(nirmsNumberAnswer) =>
+            val list = SummaryListViewModel(
+              rows = Seq(
+                Some(HasNirmsSummary.row(value = true, CheckMode)),
```

Comment:
Hardcoding true is safe since it avoids url-hacking issues where a user could enter false, submit, and then url hack to cya page without confirming their hasnirms as false. Only the nirms number is altered on submit, if the user enters false and continues they go through the hasnirms removal path and is handled there seperately. No worries about any bugs from me after manual testing.